Sources/XMTP/Conversations.swift

```diff
@@ -236,14 +236,27 @@ public actor Conversations {
         }
 
         newConversations
-            .filter { $0.peerAddress != client.address }
+            .filter { $0.peerAddress != client.address && isValidTopic(topic: $0.topic) }
             .forEach { conversationsByTopic[$0.topic] = $0 }
 
         // TODO(perf): use DB to persist + sort
         return conversationsByTopic.values.sorted { a, b in
             a.createdAt < b.createdAt
         }
     }
+    
+    public func isValidTopic(topic: String) -> Bool {
+        let regex = "^[\\x00-\\x7F]+$"
+        let initialIndex = String.Index(utf16Offset: 8, in: topic)
+        let finalIndex = topic.lastIndex(of: "/")
+        if finalIndex != nil {
+            let unwrappedTopic = String(topic[initialIndex..<finalIndex!])
+            let indices = unwrappedTopic.startIndex..<unwrappedTopic.endIndex
+            let resultMatch = unwrappedTopic.range(of: regex, options: .regularExpression)
+            return indices == resultMatch
+        }
```

Comment:
I think this is why the linter is failing with `error: Force Unwrapping Violation: Force unwrapping should be avoided (force_unwrapping)` I also don't think this is the most swifty way to do this but @nakajima might now better.

```suggestion
        if let finalIndex = topic.lastIndex(of: "/") { 
            let unwrappedTopic = String(topic[initialIndex..<finalIndex])
            let indices = unwrappedTopic.startIndex..<unwrappedTopic.endIndex
            let resultMatch = unwrappedTopic.range(of: regex, options: .regularExpression)
            return indices == resultMatch
        }
```